drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api/events");
```

Comment:
I would drop the 

    var client = new HydraClient(); 

here and perhaps introduce a separate page before this that describes following links.